package.json

```diff
@@ -6,6 +6,8 @@
   "repository": "https://github.com/ilios/lti-server",
   "scripts": {
     "preinstall": "npx only-allow pnpm",
+    "lint": "eslint . --cache",
+    "lint:fix": "eslint . --fix",
     "test": "mocha && eslint . ",
```

Comment:
Probably a good idea to pull the `eslint .` out of this line so we don't double lint on each test.

Reply:
```suggestion
    "test": "mocha",
```